<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="WorkflowMarkupSerializationException.xml" source-language="en-US" target-language="ru-RU">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac5fb2bf7810cda9c50cfa8bbdd531cff8b3ff624ff.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fb2bf7810cda9c50cfa8bbdd531cff8b3ff624ff</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException">
          <source>The exception that is thrown by methods that perform serialization and deserialization.</source>
          <target state="translated">Исключение, вызываемое методами, выполняющими сериализацию и десериализацию.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException">
          <source><ph id="ph1">&lt;xref:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException&gt;</ph> includes information about the line and column locations that caused the exception to be thrown.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException&gt;</ph> содержит сведения о причине исключения — строку и номер символа в строке.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" uid="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException" /&gt;</ph> class.</source>
          <target state="translated">Инициализирует новый экземпляр класса <ph id="ph1">&lt;see cref="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" uid="M:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException" /&gt;</ph> class.</source>
          <target state="translated">Инициализирует новый экземпляр класса <ph id="ph1">&lt;see cref="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" uid="M:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.#ctor(System.String)">
          <source>A string that contains message that describes the error.</source>
          <target state="translated">Строка, содержащая сообщение, описывающее ошибку.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" uid="M:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException" /&gt;</ph> class by using the specified error message.</source>
          <target state="translated">Инициализирует новый экземпляр <ph id="ph1">&lt;see cref="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException" /&gt;</ph> с помощью заданного сообщения об ошибке.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" uid="M:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>A <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> that contains the serialized object data about the <ph id="ph2">&lt;see cref="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException" /&gt;</ph> being thrown.</source>
          <target state="translated">Класс <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph>, содержащий данные сериализованного объекта о вызываемом исключении <ph id="ph2">&lt;see cref="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" uid="M:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>A <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> that contains the context information about the source or destination of the bits the serializer was using.</source>
          <target state="translated">Класс <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph>, содержащий сведения контекста об источнике или назначении битов, использованных сериализатором.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" uid="M:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException" /&gt;</ph> class by using serialized data.</source>
          <target state="translated">Инициализирует новый экземпляр класса <ph id="ph1">&lt;see cref="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException" /&gt;</ph>, используя сериализованные данные.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="M:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.#ctor(System.String,System.Exception)">
          <source>A string that contains message that describes the error.</source>
          <target state="translated">Строка, содержащая сообщение, описывающее ошибку.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.#ctor(System.String,System.Exception)">
          <source>The exception that is the cause of the <ph id="ph1">&lt;see cref="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException" /&gt;</ph>.</source>
          <target state="translated">Исключение, послужившее причиной <ph id="ph1">&lt;see cref="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="M:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.#ctor(System.String,System.Exception)">
          <source>If <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> is not a null reference (<ph id="ph1">&lt;see langword="Nothing" /&gt;</ph> in Visual Basic), the current exception is raised in a catch block that handles the inner exception.</source>
          <target state="translated">Если параметр <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> не является пустой ссылкой (<ph id="ph1">&lt;see langword="Nothing" /&gt;</ph> в Visual Basic), то текущее исключение возникает в блоке catch, обрабатывающем внутреннее исключение.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" uid="M:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException" /&gt;</ph> class by using an error message and an exception.</source>
          <target state="translated">Выполняет инициализацию нового экземпляра класса <ph id="ph1">&lt;see cref="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException" /&gt;</ph>, используя сообщение об ошибке и исключение.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" uid="M:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.#ctor(System.String,System.Int32,System.Int32)">
          <source>A string that contains message that describes the error.</source>
          <target state="translated">Строка, содержащая сообщение, описывающее ошибку.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" uid="M:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.#ctor(System.String,System.Int32,System.Int32)">
          <source>The line number in the object or XOML file that caused the exception to be thrown.</source>
          <target state="translated">Номер строки в объекте или файле XOML, послуживших причиной возникновения исключения.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" uid="M:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.#ctor(System.String,System.Int32,System.Int32)">
          <source>The column position in the object or XOML file that caused the exception to be thrown.</source>
          <target state="translated">Позиция столбца в объекте или файле XOML, послуживших причиной возникновения исключения.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" uid="M:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.#ctor(System.String,System.Int32,System.Int32)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException" /&gt;</ph> class by using an error message and the line and column numbers that caused the exception to be thrown.</source>
          <target state="translated">Инициализирует новый экземпляр класса <ph id="ph1">&lt;see cref="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException" /&gt;</ph> с помощью сообщения об ошибке и номеров строки и столбца, послуживших причиной возникновения исключения.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" uid="M:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.#ctor(System.String,System.Exception,System.Int32,System.Int32)">
          <source>A string that contains message that describes the error.</source>
          <target state="translated">Строка, содержащая сообщение, описывающее ошибку.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" uid="M:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.#ctor(System.String,System.Exception,System.Int32,System.Int32)">
          <source>The exception that is the cause of the <ph id="ph1">&lt;see cref="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException" /&gt;</ph>.</source>
          <target state="translated">Исключение, послужившее причиной <ph id="ph1">&lt;see cref="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" uid="M:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.#ctor(System.String,System.Exception,System.Int32,System.Int32)">
          <source>If <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> is not a null reference (<ph id="ph1">&lt;see langword="Nothing" /&gt;</ph> in Visual Basic), the current exception is raised in a catch block that handles the inner exception.</source>
          <target state="translated">Если параметр <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> не является пустой ссылкой (<ph id="ph1">&lt;see langword="Nothing" /&gt;</ph> в Visual Basic), то текущее исключение возникает в блоке catch, обрабатывающем внутреннее исключение.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" uid="M:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.#ctor(System.String,System.Exception,System.Int32,System.Int32)">
          <source>The line number in the object or XOML file that caused the exception to be thrown.</source>
          <target state="translated">Номер строки в объекте или файле XOML, послуживших причиной возникновения исключения.</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" uid="M:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.#ctor(System.String,System.Exception,System.Int32,System.Int32)">
          <source>The column position in the object or XOML file that caused the exception to be thrown.</source>
          <target state="translated">Позиция столбца в объекте или файле XOML, послуживших причиной возникновения исключения.</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" uid="M:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.#ctor(System.String,System.Exception,System.Int32,System.Int32)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException" /&gt;</ph> class by using an error message, an exception, and the line and column numbers that caused the exception to be thrown.</source>
          <target state="translated">Инициализирует новый экземпляр класса <ph id="ph1">&lt;see cref="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException" /&gt;</ph> с помощью сообщения об ошибке, исключения и номеров строки и столбца, послуживших причиной возникновения исключения.</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" uid="M:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>A <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> that contains the serialized object data about the <ph id="ph2">&lt;see cref="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException" /&gt;</ph> being thrown.</source>
          <target state="translated">Класс <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph>, содержащий данные сериализованного объекта о вызываемом исключении <ph id="ph2">&lt;see cref="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" uid="M:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>A <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> that contains the context information about the source or destination of the bits the serializer was using.</source>
          <target state="translated">Класс <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph>, содержащий сведения контекста об источнике или назначении битов, использованных сериализатором.</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve" uid="M:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Sets the specified <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> with information about the <ph id="ph2">&lt;see cref="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException" /&gt;</ph>.</source>
          <target state="translated">Задает в указанном объекте <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> сведения об исключении <ph id="ph2">&lt;see cref="T:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source><ph id="ph1">&lt;xref:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.GetObjectData%2A&gt;</ph> sets a <ph id="ph2">&lt;xref:System.Runtime.Serialization.SerializationInfo&gt;</ph> with all the exception object data targeted for serialization.</source>
          <target state="translated">Метод <ph id="ph1">&lt;xref:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.GetObjectData%2A&gt;</ph> загружает в объект <ph id="ph2">&lt;xref:System.Runtime.Serialization.SerializationInfo&gt;</ph> все данные объекта исключения, предназначенные для сериализации.</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>During de-serialization, the exception is reconstituted from the <ph id="ph1">&lt;xref:System.Runtime.Serialization.SerializationInfo&gt;</ph> transmitted over the stream.</source>
          <target state="translated">В процессе десериализации исключение воссоздается из класса <ph id="ph1">&lt;xref:System.Runtime.Serialization.SerializationInfo&gt;</ph>, переданного в потоке.</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve" uid="P:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.LineNumber">
          <source>Gets the line number in the object or XOML file that caused the exception to be thrown.</source>
          <target state="translated">Возвращает номер строки в объекте или файле XOML, послуживших причиной возникновения исключения.</target>       </trans-unit>
        <trans-unit id="130" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.LineNumber">
          <source>An integer that represents the line number on which the exception was thrown.</source>
          <target state="translated">Целое число, представляющее номер строки, в которой возникло исключение.</target>       </trans-unit>
        <trans-unit id="131" translate="yes" xml:space="preserve" uid="P:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.LinePosition">
          <source>Gets the column position in the object or XOML file that caused the exception to be thrown.</source>
          <target state="translated">Возвращает позицию столбца в объекте или файле XOML, послуживших причиной возникновения исключения.</target>       </trans-unit>
        <trans-unit id="132" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.LinePosition">
          <source>An integer that represents the column position that caused the exception to be thrown.</source>
          <target state="translated">Целое число, представляющее позицию столбца, послужившую причиной возникновения исключения.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>